src/Plugin/Field/FieldFormatter/AuthorsReferenceFormatter.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\Plugin\Field\FieldFormatter\EntityReferenceFormatterBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Plugin implementation of the Author reference fields formatter.
+ *
+ * @FieldFormatter(
+ *   id = "authors_reference_formatter",
```

Comment:
```suggestion
 *   id = "oe_content_authors_reference_formatter",
```
Prefix has to be `oe_`